scala-test-suite/src/test/resources/2.13.13/BlacklistedTests.txt

```diff
@@ -0,0 +1,247 @@
+scala/tools/xsbt/TestCallback.scala
+scala/util/ChainingOpsTest.scala
+
+## Do not link
+scala/CollectTest.scala
+scala/MatchErrorSerializationTest.scala
+scala/PartialFunctionSerializationTest.scala
+scala/lang/stringinterpol/StringContextTest.scala
+scala/collection/IterableTest.scala
+scala/collection/IteratorTest.scala
+scala/collection/NewBuilderTest.scala
+scala/collection/SeqViewTest.scala
+scala/collection/SetMapConsistencyTest.scala
+scala/collection/SetMapRulesTest.scala
+scala/collection/Sizes.scala
+scala/collection/ViewTest.scala
+scala/collection/concurrent/TrieMapTest.scala
+scala/collection/convert/EqualsTest.scala
+scala/collection/convert/JConcurrentMapWrapperTest.scala
+scala/collection/convert/WrapperSerializationTest.scala
+scala/collection/immutable/ChampMapSmokeTest.scala
+scala/collection/immutable/ChampSetSmokeTest.scala
+scala/collection/immutable/LazyListGCTest.scala
+scala/collection/immutable/LazyListLazinessTest.scala
+scala/collection/immutable/ListTest.scala
+scala/collection/immutable/SerializationTest.scala
+scala/collection/immutable/StreamTest.scala
+scala/collection/immutable/StringLikeTest.scala
+scala/collection/immutable/VectorTest.scala
+scala/collection/mutable/AnyRefMapTest.scala
+scala/collection/mutable/ArrayBufferTest.scala
+scala/collection/mutable/ListBufferTest.scala
+scala/collection/mutable/OpenHashMapTest.scala
+scala/collection/mutable/PriorityQueueTest.scala
+scala/collection/mutable/SerializationTest.scala
+scala/concurrent/FutureTest.scala
+scala/concurrent/duration/SerializationTest.scala
+scala/concurrent/impl/DefaultPromiseTest.scala
+scala/io/SourceTest.scala
+scala/jdk/AccumulatorTest.scala
+scala/jdk/DurationConvertersTest.scala
+scala/jdk/FunctionConvertersTest.scala
+scala/jdk/OptionConvertersTest.scala
+scala/jdk/StepperConversionTest.scala
+scala/jdk/StepperTest.scala
+scala/jdk/StreamConvertersTest.scala
+scala/jdk/StreamConvertersTypingTest.scala
+scala/math/OrderingTest.scala
+scala/runtime/ScalaRunTimeTest.scala
+scala/sys/env.scala
+scala/sys/process/ParserTest.scala
+scala/sys/process/PipedProcessTest.scala
+scala/sys/process/ProcessBuilderTest.scala
+scala/sys/process/ProcessTest.scala
+scala/tools/testkit/AssertUtilTest.scala
+scala/util/PropertiesTest.scala
+scala/util/SpecVersionTest.scala
+scala/util/SystemPropertiesTest.scala
+
+## Tests fail
+
+# Reflection
+scala/reflect/ClassTagTest.scala
+
+# Require strict-floats
+scala/math/BigDecimalTest.scala
+
+# Tests passed but are too slow (timeouts)
+scala/collection/immutable/ListSetTest.scala
+scala/util/SortingTest.scala
+
+# Whitebox testing of ArrayBuilder.ofUnit
+scala/collection/mutable/ArrayBuilderTest.scala
+
+# Relies on undefined behavior
+scala/collection/MapTest.scala
+scala/collection/StringOpsTest.scala
+scala/collection/StringParsersTest.scala
+scala/collection/convert/CollectionConvertersTest.scala
+scala/collection/convert/MapWrapperTest.scala
+scala/collection/immutable/NumericRangeTest.scala
```

Comment:
`NumericRangeTest.scala` is new here. It relies on catching `ClassCastException`.